templates/repo/settings/webhook/history.tmpl

```diff
@@ -1,9 +1,13 @@
+{{$isNew:=or .PageIsSettingsHooksNew .PageIsAdminDefaultHooksNew .PageIsAdminSystemHooksNew}}
 {{if .PageIsSettingsHooksEdit}}
 	<h4 class="ui top attached header">
 		{{.locale.Tr "repo.settings.recent_deliveries"}}
 		{{if .Permission.IsAdmin}}
 			<div class="ui right">
-				<button class="ui teal tiny button" id="test-delivery" data-tooltip-content="{{.locale.Tr "repo.settings.webhook.test_delivery_desc"}}" data-link="{{.Link}}/test" data-redirect="{{.Link}}">{{.locale.Tr "repo.settings.webhook.test_delivery"}}</button>
+				<!-- the button is wrapped with a span because the tooltip doesn't show on hover if we put data-tooltip-content directly on the button -->
```

Comment:
I don't know why the tooltip doesn't show on a disabled `button` so I wrapped it in a `span`

Reply:
I think this is per browser design where disabled elements do not emit mouse events and such:

https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/disabled#overview
https://github.com/atomiks/tippyjs/issues/286

> Often browsers grey out such controls and it won't receive any browsing events, like mouse clicks or focus-related ones.

Reply:
Thought I see we don't actually set `disabled` attribute, only `disabled` class, so it should still work unless the CSS sets options like `inert` or `pointer-events: none` on it.

Reply:
Actually yes, I see `.ui.disabled.button` sets `pointer-events: none` so that is likely why tippy does not work. I guess a wrapper is a ok-ish solution.

Reply:
That's even the [official solution](https://github.com/atomiks/tippyjs/issues/104), apparently.

Reply:
We do have a little bit more control because our CSS sets the `pointer-events`, but I can't recommend enabling them because that would make the button clickable, so it would need additional JS to ensure it's unclickable.